banner_react/src/Banner.js

```diff
@@ -61,11 +59,11 @@ class Banner extends Component {
     transitioning = () => {
         this.props.whenTransition();
         this.myTimer = setTimeout(
-            this.transitioning, this.state.transTime / 30
+            this.transitioning, this.state.transTime / 31
```

Comment:
I want it to be 60 fps
```suggestion
            this.transitioning, this.state.transTime / 60
```